main.js

```diff
@@ -3,11 +3,30 @@ var buttonRow = document.getElementById("buttonRow")
 var category = '';
 
 buttonRow.addEventListener("click", function(event){
+  if (event.target.id === "studyButton") {
```

Comment:
Nice getting the functionality working! We may want to come back and refactor this one before the final iteration